app/policies/comment_policy.rb

```diff
@@ -11,18 +11,18 @@ def show?
 
   def create?
     if Array.wrap(record).compact.any? { |c| c.discussion.board.section == 'zooniverse' }
-      logged_in? && !locked? && writable? && of_posting_age?
```

Comment:
Do we still want the `of_posting_age?` restriction now that we have `confirmed?`